CondTools/SiPhase2Tracker/python/ConfFile_cfg.py

```diff
@@ -0,0 +1,20 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
what is the purpose of this configuration file?

Reply:
Er... I believe mkedanlzr dumps it in the python directory trying to be nice. I fear I'll have to squash commits to remove it.